test/src/main/resources/log4j.properties

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-log4j.rootLogger=DEBUG, CA
+log4j.rootLogger=ERROR, CA
```

Comment:
Is this necessary for the test? We typically need DEBUG logs when a test fails, to figure out what happened.

Reply:
No. I had changed it to reduce some output at one point and overlooked it when committing. I reverted back to DEBUG